2-mandatory/4-space-colonies.js

```diff
@@ -15,7 +15,13 @@
   
 */
 
-function getSettlers() {}
+function getSettlers(arrayOfNames) {
+  arrayOfNames.filter ( (name => {
```

Comment:
There's a similar error happening here as in formatPercentages. Take a look again at what your code is returning. 

Take this problem to your Mentor ASAP as it's simple to clear up but very important to understand.